vms/platformvm/config/execution_config.go

```diff
@@ -21,6 +21,7 @@ var DefaultExecutionConfig = ExecutionConfig{
 	ChainDBCacheSize:             2048,
 	BlockIDCacheSize:             8192,
 	FxOwnerCacheSize:             4 * units.MiB,
+	SubnetManagerCacheSize:       2048,
```

Comment:
This is pretty confusing. All of the bytes caches use `units.MiB` to be clear that it is representing bytes. All of the other values are denominating the number of entries.

This entry is interpreted as bytes - but is set as if it is in the number of entries.

I think we should either be specifying `2 * units.KiB` or we should be setting this to a different value (if the assumption was that this is denominated in entries rather than bytes)